libecs/include/pivot/ecs/Core/Scene.hxx

```diff
@@ -137,6 +140,8 @@ private:
     pivot::ecs::systems::Manager mSystemManager;
     pivot::ecs::event::Manager mEventManager;
     pivot::ecs::component::Manager::ComponentId mTagId;
+    pivot::ecs::component::Manager mCopyComponentManager;
+    EntityManager mCopyEntityManager;
```

Comment:
I would prefer having both of the manager into a separate struct.

Reply:
Oh no my bad it's a mistake ! I didn't want to push that